pkg/operator/vspherecontroller/config.go

```diff
@@ -19,7 +19,7 @@ type iniConfig struct {
 func newINIConfig(config string) (*iniConfig, error) {
 	// We ignore inline comments when reading the INI file
 	// because passwords may contain comment symbols (# and ;)
-	options := iniv1.LoadOptions{IgnoreInlineComment: true}
+	options := iniv1.LoadOptions{IgnoreInlineComment: true, PreserveSurroundedQuote: true}
```

Comment:
Why did we had to use `PreserveSurroundedQuote` here? cc @bertinatto

Reply:
It was a change from the other PR.  there was logic that was forcing this after the fact and this fixes it i thought a little cleaner since I had to fix merge conflict: 
https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/239/files#diff-bd7b198fe282f7e27754079508639882be175cc903f563fb69f2c89fc6875a2aR698